Pre Sequel Mods/VexilleSerra/Item lvl Fix.blcm

```diff
@@ -0,0 +1,30 @@
+<BLCMM v="1">
+#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!>
+	<head>
+		<type name="TPS" offline="false"/>
+	</head>
+	<body>
+		<category name="Item Lvl Fix">
+			<category name="Description">
+				<comment>Makes vendors and enemies give you on-level gear only</comment>
+			</category>
+			<category name="Code">
+				<hotfix name="ItemLvlFix" level="None">
+					<code profiles="default">set GD_Balance.RandomNumbers.Random_Minus02-Plus01 RandomVariance (bEnabled=True,bUseIntegerRandomization=True,LowerBound=(BaseValueConstant=0.000000,BaseValueAttribute=None,InitializationDefinition=None,BaseValueScaleConstant=1.000000),UpperBound=(BaseValueConstant=0.000000,BaseValueAttribute=None,InitializationDefinition=None,BaseValueScaleConstant=1.000000))</code>
+					<code profiles="default">set GD_Economy.VendingMachine.Init_VendingMachine_LootGamestageVariance RandomVariance (bEnabled=True,bUseIntegerRandomization=True,LowerBound=(BaseValueConstant=0.000000,BaseValueAttribute=None,InitializationDefinition=None,BaseValueScaleConstant=1.000000),UpperBound=(BaseValueConstant=0.000000,BaseValueAttribute=None,InitializationDefinition=None,BaseValueScaleConstant=1.000000))</code>
+					<code profiles="default">set GD_Balance.RandomNumbers.Random_Minus01-Plus01 RandomVariance (bEnabled=True,bUseIntegerRandomization=True,LowerBound=(BaseValueConstant=0.000000,BaseValueAttribute=None,InitializationDefinition=None,BaseValueScaleConstant=1.000000),UpperBound=(BaseValueConstant=0.000000,BaseValueAttribute=None,InitializationDefinition=None,BaseValueScaleConstant=1.000000))</code>
+				</hotfix>
+			</category>
+		</category>
+	</body>
+</BLCMM>
+
+#Commands:
+
+#Direct-Execute Warning:
+say WARNING: "Item Lvl Fix" must be imported into BLCMM to run properly with UCP or other mods.
+
+#Hotfixes:

```

Comment:
I don't know enough about modding to understand if this `#Hotfixes` section is applicable to TPS or not.  I can remove it if anyone has a hint.